Documentation/Signoffs/NutrientSubsystemSignoff.md

```diff
@@ -0,0 +1,104 @@
+   4. The sensor measures soil moisture with a range of 0-100% with an accuracy of ±2% within the 0-50% moisture range and ±3% within the 50-100% range and a resolution of 0.10%.
+   5. The sensor measures nitrogen levels within a range of 1-1999mg/kg with an accuracy of ±2% full scale and a resolution of 1mg/kg.
+   6. The sensor measures phosphorus levels within a range of 1-1999mg/kg with an accuracy of ±2% full scale and a resolution of 1mg/kg.
+   7. The sensor measures potassium levels within a range of 1-1999mg/kg with an accuracy of ±2% full scale and a resolution of 1mg/kg.
+   8. The sensor measures pH within a range of 3-9pH with an accuracy of ±0.3pH and a resolution of 0.01pH.
+
+- **DC to DC Converter - Nisshinbo R1210N601D-TR-FE**
+  
+   1. The converter takes an input voltage range of 0.8-8V and produces a constant output of 6V ±0.15V.
+   2. This is within the range of input voltage for the Arduino Nano 33 IoT and reduces the risk of having too little voltage for the Arduino as the battery charges dissipate.
+   3. Since the minimum input voltage of the converter is 0.8V this means that the system will be able to operate until the power supply voltage drops below 0.2V per battery which means that roughly ((1.5V-0.2V)/1.5V) = 0.867 or 86.7% of the individual battery's supply is used before needing to be replaced.
+ 
+- **Power Analysis**
+  
+    1. The power is routed through the Arduino so that the sensor may be cycled on and off as the Nano 33 IOT enters and exits sleep mode, while the Arduino is in sleep mode it does not produce a high enough voltage for any of the systems components to function so they are all essentially powered off.[4]
+    2. The Arduino will be cycled out of sleep mode for 10 seconds every 5 minutes leading to the sensors using (10s/3600s) for the milliwatt hour calculations, or 10 seconds per hour, and (5min/60min) for the average power consumption calculations from the 5 minute cycle.
+    3. The system is powered by 4 AA batteries each supplying 3500mAh and 1.5V.
+    4. The uxcell MAX485 has a current draw of <5ma in on mode and 0.3mA in sleep mode.
+    5. The CWT-SOIL-NPKPHCTH-S has a max power usage of 500mW at 24V giving us (500mw/24V) = 20.83mA current draw and has a current draw of 4mA in sleep mode [5].
+    6. The Nisshinbo R1210N601D-TR-FE uses 0.09mA at 6V.
+    8. The duty cycle of the components would be (10s / (5min * 60s)) = 0.0333 because the system is only turned on for 10 seconds every 5 minutes.
+    9. Therefore the Ardunio has an current draw of 28mA in on mode and 6mA in sleep mode.
+    12. According to the simulations below the system has a current draw of 53.92mA in on mode and 10.39mA in off mode.
+    13. When applying the duty cycle this gives us an average current consumption of (53.92mA * 0.0333) + (10.39mA * 0.0333) = 2.143mA.
```

Comment:
The calculation above is incorrect. The duty cycle should only affect the "on" time. The off time current should be multiplied by 1-duty cycle (0.9666).